pkg/publicapi/endpoint/orchestrator/job.go

```diff
@@ -114,13 +115,26 @@ func (e *Endpoint) getJob(c echo.Context) error {
 				return err
 			}
 			response.Executions = &apimodels.ListJobExecutionsResponse{
-				Executions: make([]*models.Execution, len(executions)),
+				Items: make([]*models.Execution, len(executions)),
 			}
 			for i := range executions {
-				response.Executions.Executions[i] = &executions[i]
+				response.Executions.Items[i] = &executions[i]
 			}
 		}
 	}
+
+	clientVersion := apimodels.GetClientVersion(c.Request())
+	if !(clientVersion.GreaterThan(version.V1_3_2) || clientVersion.Equal(version.Development) || clientVersion.Prerelease() != "") {
```

Comment:
Since we have been encoding the client version for a while, we shouldn't only return old format if the version is known and it is older than 1.3.2. For development and unknown versions we should return the latest format

Reply:
So, we do return the new version if the client's version is Development:
```shell
curl -s -H "X-Bacalhau-Git-Version: v0.0.0-xxxxxxx" 0.0.0.0:1234/api/v1/orchestrator/jobs | jq '{JobsCount: (if .Jobs == null then 0 else .Jobs | length end), ItemsCount: .Items | length}'
{
  "JobsCount": 0,
  "ItemsCount": 4
}
```
But don't if the client's version is unknown:
(with header)
```shell
curl -s -H "X-Bacalhau-Git-Version: v0.0.0" 0.0.0.0:1234/api/v1/orchestrator/jobs | jq '{JobsCount: (if .Jobs == null then 0 else .Jobs | length end), ItemsCount: .Items | length}'
{
  "JobsCount": 4,
  "ItemsCount": 0
}
```
(without header)
```
curl -s  0.0.0.0:1234/api/v1/orchestrator/jobs | jq '{JobsCount: (if .Jobs == null then 0 else .Jobs | length end), ItemsCount: .Items | length}'
{
  "JobsCount": 4,
  "ItemsCount": 0
}
```
Meaning the requested change here is to use the new format if the client version is not set/unknown?